arrebol/worker/whitelist/whitelist.go

```diff
@@ -0,0 +1,60 @@
+package whitelist
+
+import (
+	"bufio"
+	"log"
+	"os"
+)
+
+const (
+	WhiteListPath = "WHITE_LIST_PATH"
+)
+
+type WhiteList interface {
+	Contains(id string) (bool)
+}
+
+type FileWhiteList struct {
+	sourceFile string
+}
+
+func NewFileWhiteList() WhiteList {
+	return &FileWhiteList{sourceFile: os.Getenv(WhiteListPath)}
+}
+
+func (l *FileWhiteList) Contains(workerId string) (bool) {
+	ids, err := l.loadSourceFile()
```

Comment:
you could have a ids filed in the whitelist struct, it will be set in "construction time", so you don't have to load it every time contains is called. In the NewWhiteList you must call loadSourceFIle and assign it to its field, what do you think?

Reply:
Done.